### PR TITLE
add description of how to scratch build nightly packages

### PR DESCRIPTION
**Pull request by komidore64**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [ ] 1.20
* [ ] 1.19
* [ ] 1.18

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -59,6 +59,16 @@ than mock (above).
...
 
+However, `scratch` will not work for nightly packages (i.e. - packages that
+don't have sources tracked in foreman-packages). The process for these is a bit
+different. Build the source files externally, then point `obal nightly` at
+them.
+
+    obal nightly foreman-installer \
+        --scratch \
+        --source ~/workspace/foreman-installer/pkg/foreman-installer-1.20.0-develop.tar.bz2 \
+        --githash 3ff078d12560c3f7ec55dad6bc70dd87dd7855a0
```
Comment (ehelms) [inline README.md]: Is this part required? Might be worth mentioning acquiring this from somewhere if it is especially if I got my source from somewhere else.

File: README.md [hunk 2]
```diff
@@ -59,6 +59,16 @@ than mock (above).
 
     obal scratch PACKAGE
 
+However, `scratch` will not work for nightly packages (i.e. - packages that
```
Comment (ekohl) [inline README.md]: This must work for our PR testing to work.

File: README.md (hunk 1 above)
Comment (komidore64) [reply README.md]: the git has is require. it's needed for the `<timestamp>git<hash>` string that appended to the release.

File: README.md (hunk 2 above)
Comment (komidore64) [reply README.md]: hrmmmmmmm. i need to think about this.

**Comment by ehelms** [comment]
Closing due to inactivity

**Pull request closed without merging**